src/Models/Tag.php

```diff
@@ -15,18 +15,20 @@
 
 class Tag extends BaseModel implements Sortable
 {
-    use SortableTrait, HasSlug, HasCreator, HasUpdater, HasPackageFactory;
+    use SortableTrait, HasCreator, HasUpdater, HasPackageFactory;
 
     protected $guarded = ['id'];
```

Comment:
$guarded can be removed since it is in BaseModel

Reply:
34e87fada80772254c1346a34cceee79a0ae3199